client/me/purchases/manage-purchase/index.tsx

```diff
@@ -679,6 +723,7 @@ class ManagePurchase extends Component<
 			>
 				<MaterialIcon icon="delete" className="card__icon" />
 				{ text }
+				{ this.renderRefundText() }
```

Comment:
I think this one should be removed.  In practice it doesn't display anything (because the `RemovePurchase` component should only ever appear when the purchase isn't refundable in the first place), but if for some reason it does appear, we wouldn't want to tell people that a refund is available here, because going through the "Remove" flow (unlike the "Cancel" flow) will never trigger a refund.